### PR TITLE
Enhancement: Keep packages sorted in composer.json

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] keeps packages sorted in `composer.json`

💁‍♂️ For reference, see https://getcomposer.org/doc/06-config.md#sort-packages.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
         }
     ],
     "require": {
-      "ext-curl": "*",
-      "lib-curl": "*",
-      "php": ">=5.3.0"
+        "php": ">=5.3.0",
```
Comment (hush-hush) [inline composer.json]: Why change the require order ? They no longer seem to be sorted by alphabetical order.

Comment (localheinz) [reply composer.json]: That’s true, according to https://github.com/composer/composer/blob/c917865fe9f62fd0a7fa6db910597ec4259fc7bf/src/Composer/Json/JsonManipulator.php#L110-L146, requirements are sorted by type first, then alphabetically.

Comment (localheinz) [reply composer.json]: If you are interested, take a look at

- https://localheinz.com/blog/2018/01/15/normalizing-composer.json/

🤓

Comment (hush-hush) [reply composer.json]: Thanks for the links !

**Comment by localheinz** [comment]
Thank you, @hush-hush!

**Pull request merged**